R/qtr.R

```diff
@@ -0,0 +1,139 @@
+#' @title Find a quarter to a date
+#'
+#' @description
+#'
+#' The qtr functions take a date input and calculate the relevant
+#' quarter-related value from it.
+#'
+#' \itemize{
+#' \item \code{qtr_year} returns the current quarter and year in either short or
+#' long format.
+#'
+#' \item \code{qtr_end} returns the last month in the quarter in either short or
+#' long format.
+#'
+#' \item \code{qtr_prev} returns the previous quarter in either short or long
+#' format.
+#' }
+#'
+#' @param date A date supplied with \code{Date} class.
+#' @param format A \code{character} string specifying the format the quarter
+#' should be displayed in. Valid options are `short` (Jan-Mar 2018) and `long`
+#' (January to March 2018).
```

Comment:
I think we should probably also say what the default behaviour of `format` is. All the documentation lists 'short' before 'long', and I think people might assume that 'short' is the default as a result if they don't know otherwise.